css/flag-icon.css

```diff
@@ -1512,6 +1512,24 @@
 .flag-icon-zw.flag-icon-squared {
   background-image: url(../flags/1x1/zw.svg);
 }
+.flag-icon-ac {
+  background-image: url(../flags/4x3/ac.svg);
+}
+.flag-icon-ac.flag-icon-squared {
+  background-image: url(../flags/1x1/ac.svg);
+}
+.flag-icon-cp {
+  background-image: url(../flags/4x3/cp.svg);
+}
+.flag-icon-cp.flag-icon-squared {
+  background-image: url(../flags/1x1/cp.svg);
+}
+.flag-icon-ea {
+  background-image: url(../flags/4x3/ea.svg);
+}
+.flag-icon-ea.flag-icon-squared {
+  background-image: url(../flags/1x1/aea.svg);
```

Comment:
is that a typo? `aea.svg`

Reply:
Yep, good catch. What's the overall view on the PR? I can push a fix if it's gonna move forward